requirements.txt

```diff
@@ -1,6 +1,15 @@
-astroid==1.3.4
-logilab-common==0.63.2
-nose==1.3.4
-pep8==1.6.2
-pylint==1.4.1
-six==1.9.0
+astroid==1.6.6 ; python_version < '3'
+astroid==2.4.1 ; python_version >= '3'
+isort==4.3.21
+lazy-object-proxy==1.4.3
+logilab-common==1.4.4 ; python_version < '3'
+logilab-common==1.6.1 ; python_version >= '3'
+mccabe==0.6.1
+nose==1.3.7
+pycodestyle==2.6.0
+pylint==1.9.5 ; python_version < '3'
+pylint==2.5.2 ; python_version >= '3'
+six==1.15.0
+toml==0.10.1
+typed-ast==1.4.1 ; python_version >= '3'
+wrapt==1.12.1
```

Comment:
Where do all the new dependencies come from?

`pip freeze` (of a python 3 `virtualenv`)?

Reply:
I would say that we only need:

```
nose==1.3.7
pycodestyle==2.5.0
pylint==1.9.5
```

the (new) python version syntax looks nice `; python_version >= '3'`